### PR TITLE
Make artificial viscosity documentation more general

### PR DESCRIPTION
**Pull request by thomasgibson**
Since the AV implementation doesn't specifically rely on anything Euler-specific, let the docs reflect that

### CODE REVIEW

**Comment by thomasgibson** [comment]
@MTCam @w-hagen

File: mirgecom/artificial_viscosity.py [hunk 1]
```diff
@@ -1,27 +1,31 @@
-r""":mod:`mirgecom.artificial_viscosity` Artificial viscocity for Euler.
+r""":mod:`mirgecom.artificial_viscosity` Artificial viscosity for hyperbolic
+conservation laws.
```
Comment (w-hagen) [inline mirgecom/artificial_viscosity.py]: I think docstyle will complain about this? I think this has to be a one-liner?

Comment (thomasgibson) [reply mirgecom/artificial_viscosity.py]: Done in https://github.com/w-hagen/mirgecom/pull/18/commits/9b1dcc89545d8b489fb7aa841595578d1a57fc84

**Comment by w-hagen** [comment]
Looks good, Thanks!

**Pull request merged**